resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/ClientResponse.java

```diff
@@ -305,6 +306,8 @@ else if (bufferedEntity == null)
             {
 
             }
+            // record error details in server.log
+            LogMessages.LOGGER.clientSideProcessingFailure(e);

```

Comment:
I guess this will conflict with the pull request for RESTEASY-1991.

Reply:
@marekkopecky Why don't you address the test questions, since you wrote the test.

Reply:
There might be a conflict with RESTEASY-1991.  That will have to be addressed during the merges.

Reply:
@rsearls I don't see my original commit, so it is not clear, what is my original code, and what is your addition. Can you please use my commit in this PR, and add your commit with your updates?